neurons/remote_config.py

```diff
@@ -85,6 +88,35 @@ def stop_update_thread(self):
         self.stop_event.set()
         self.thread.join()
 
+    def fetch_inmemory_hotkeys(self, current_netuid: int=15):        
+        finney_subtensor = bt.subtensor(network='finney')     
```

Comment:
can we use our subtensor? finney is public one and its not under our control

Reply:
are you planning to use  the subtensor miners are connected

Reply:
this have to also work on our private network, so we can not use finney in that case

Reply:
we can use local subtensor